windows/bleachbit.nsi

```diff
@@ -441,7 +440,14 @@ FunctionEnd
 UninstallText "BleachBit will be uninstalled from the following folder.  Click Uninstall to start the uninstallation.  WARNING: The uninstaller completely removes the installation directory including any files (such as custom cleaners) that you may have added or changed."
 
 Section "Uninstall"
-    RMDir /r "$INSTDIR"
+    Delete $INSTDIR\share\fonts\segoeui.ttf 
+    Delete $INSTDIR\share\fonts\tahoma.ttf
+    RMDir $INSTDIR\share\fonts
+    
+    Delete "$INSTDIR\uninstall.exe"
```

Comment:
How about removing `uninstall.exe` after SHRED_REGEX_KEY? In case something goes wrong, this could allow a chance to try again.